server/models/UserModel/index.js

```diff
@@ -58,6 +62,32 @@ const UserModel = {
 
     return value
   },
+  hasNoAdmin: async (db) => {
+    const userCt = await db
+      .collection(collections.users)
+      .countDocuments({ role: "admin" })
+    return userCt === 0
+  },
+  createFirstAdmin: async (db, dataDb) => {
+    const reset_key = crypto.randomBytes(32).toString('hex')
+    const access = await StudiesModel.all(dataDb)
```

Comment:
I thought we had migrated from the 2 dbs to 1?

Reply:
Did we actually run that migration though? My setup locally is using two Mongo databases, but I can look into it.